pkg/services/govmomi/create_test.go

```diff
@@ -38,6 +40,7 @@ func TestCreate(t *testing.T) {
 		t.Fatal(err)
 	}
 	model.Service.TLS = new(tls.Config)
+	model.Service.RegisterEndpoints = true
```

Comment:
This line and `_ "github.com/vmware/govmomi/vapi/simulator"` are needed to make simulator register rest session endpoint or it returns an error - `unable to create rest client: POST https://127.0.0.1:63223/rest/com/vmware/cis/session: 404 Not Found`